runner/ansible/vars/azure_env.yml

```diff
@@ -24,3 +24,50 @@ expected:
   "1.4.1.pcmk_reboot_timeout": "900"
   "1.4.1.power_timeout": "240"
   "1.4.2": "true"
+  "2.1.1.resource_stickiness": "1000"
+  "2.1.1.migration_threshold": "5000"
+  "2.1.2.monitor_interval": "10"
+  "2.1.2.monitor_timeout": "600"
+  "2.1.2.start_interval": "0"
+  "2.1.2.start_timeout": "600"
+  "2.1.2.stop_interval": "0"
+  "2.1.2.stop_timeout": "300"
+  "2.1.3.master_interval": "60"
+  "2.1.3.master_timeout": "700"
+  "2.1.3.slave_interval": "61"
+  "2.1.3.slave_timeout": "700"
+  "2.1.3.start_interval": "0"
+  "2.1.3.start_timeout": "3600"
+  "2.1.3.stop_interval": "0"
+  "2.1.3.stop_timeout": "3600"
+  "2.1.3.promote_interval": "0"
+  "2.1.3.promote_timeout": "3600"
+  "2.1.3.nvpair_PREFER_SITE_TAKEOVER": "true"
+  "2.1.3.nvpair_AUTOMATED_REGISTER": "false"
+  "2.1.3.nvpair_DUPLICATE_PRIMARY_TIMEOUT": "7200"
+  "2.1.4.monitor_interval": "10"
+  "2.1.4.monitor_timeout": "20"
+  "2.1.5.nvpair_resource_stickiness": "0"
+  "2.1.6.rsc_colocation_score": "4000"
+  "2.1.6.rsc_colocation_rsc": "g_ip"
+  "2.1.6.rsc_colocation_rsc_role": "Started"
+  "2.1.6.rsc_colocation_with_rsc": "msl_SAPHana"
+  "2.1.6.rsc_colocation_with_rsc_role": "Master"
+  "2.1.6.rsc_order_kind": "Optional"
+  "2.1.6.rsc_order_first": "cln_SAPHanaTopology"
+  "2.1.6.rsc_order_then": "msl_SAPHana"
+  "2.1.7.provider": "provider = SAPHanaSR"
```

Comment:
Can we change the next for lines to only contain the value, and not the key?
It requires to change the test of course